sources/presentation/Stride.Core.Quantum/NodeAccessor.cs

```diff
@@ -63,7 +63,7 @@ public void UpdateValue(object value)
             }
             else
             {
-                ((IMemberNode)Node).Update(value);
```

Comment:
Maybe it would be better to make use of the `IsMember` and `IsItem` properties in this method to ensure the cast is doable before calling update? Also, what is the type of Node when you see the error, or is it null? If there's another type, maybe there should be another case for it. If it's null, would be good to understand why and should the code allow for that.

Reply:
I would also prefer we investigate the reason of this mistyping.

Reply:
You are right, those properties have the info already.
The node when it crashes is the List.
```Node = {{Node: Object List`1 = [System.Collections.Generic.List`1[Stride.Engine.Entity]]}}```